tests/2017-05-15-preview/Microsoft.Automation.tests.json

```diff
@@ -0,0 +1,4 @@
+{
+    "tests": [
+    ]
```

Comment:
add tests here?

Reply:
The `2017-05-15-preview` version is currently being added and there are no tests for the same right now. I'll work with the team to populate the tests. I'll remove this file and once the tests are in, put the file back in.

Reply:
Added the tests here.